lnbits/core/views/public_api.py

```diff
@@ -40,19 +40,18 @@ async def api_public_payment_longpolling(payment_hash):
 
     response = None
 
-    async def payment_info_receiver(cancel_scope):
-        async for payment in payment_queue.get():
+    async def payment_info_receiver():
+        for payment in await payment_queue.get():
```

Comment:
- hmm, are these two `for` syntaxes equivalent. I think so... 
- @dni why is one better than the other?